inc/saivlan.h

```diff
@@ -452,6 +466,28 @@ typedef enum _sai_vlan_member_attr_t
      */
     SAI_VLAN_MEMBER_ATTR_VLAN_TAGGING_MODE,
 
+    /**
+     * @brief Designator Forwarding Mode
+     *
+     * Valid only for Ethernet Segment Ports - EVPN Multihoming feature
+     *
+     * @type sai_vlan_df_mode_t
+     * @flags CREATE_AND_SET
+     * @default SAI_VLAN_DF_MODE_IGNORE
+     */
+    SAI_VLAN_MEMBER_ATTR_DF_MODE,
+
+    /**
+     * @brief Split Horizon List ID
+     *
+     * Valid only for Ethernet Segment Ports - EVPN Multihoming feature
```

Comment:
what is the PORT attribute to define whether the port is Ethernet segment PORT or not...?